sample-app/ios/sample-app/AppDelegate.m

```diff
@@ -27,6 +28,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
+  NetLogger* nl = [NetLogger sharedManager];
+  [nl  show:ALIGN_RIGHT];
```

Comment:
Nope, we don't want this in the sample app

Reply:
👍